pkgs/top-level/all-packages.nix

```diff
@@ -15629,7 +15629,9 @@ in
 
   iputils = callPackage ../os-specific/linux/iputils { };
 
-  iptables = callPackage ../os-specific/linux/iptables { };
+  iptables = iptables-legacy;
+  iptables-legacy = callPackage ../os-specific/linux/iptables { };
```

Comment:
I would not call it `legacy` because the current iptables seems no where near deprecation at the moment.

Reply:
The Netfilter project, has agreed to consider iptables as a legacy tool.
https://ral-arturo.org/2018/06/16/nfws2018.html

Reply:
Fair enough.